storagemarket/types.go

```diff
@@ -192,3 +194,26 @@ type ProviderDealState struct {
 	DealID        abi.DealID
 	FastRetrieval bool
 }
+
+// ListDealsPageParams configures the params which will be used to paginate & filter the deals that are returned by the
+// list deals commands.
+type ListDealsPageParams struct {
+	// CreationTimePageOffset is the minimum creation time of the deals which will be included in the deal list page.
```

Comment:
The comment is not quite right - the deals included in the deal list page start from _after_ CreationTimePageOffset right?

Reply:
Fixed.